docs/cn/Clients-S3.md

```diff
@@ -0,0 +1,359 @@
+# curl -i -X PUT -T "LICENSE" http://localhost:39999/api/v1/s3/testbucket/key2
+# curl -i -X PUT -T "LICENSE" http://localhost:39999/api/v1/s3/testbucket/key3
+# curl -i -X GET http://localhost:39999/api/v1/s3/testbucket\?max-keys\=2
+HTTP/1.1 200 OK
+Date: Tue, 29 Aug 2017 22:40:45 GMT
+Content-Type: application/xml
+Content-Length: 537
+Server: Jetty(9.2.z-SNAPSHOT)
+ 
+<ListBucketResult xmlns=""><Name>/testbucket</Name><Prefix/><ContinuationToken/><NextContinuationToken>key3</NextContinuationToken><KeyCount>2</KeyCount><MaxKeys>2</MaxKeys><IsTruncated>true</IsTruncated><Contents><Key>key1</Key><LastModified>2017-08-29T15:40:42.213Z</LastModified><ETag></ETag><Size>26847</Size><StorageClass>STANDARD</StorageClass></Contents><Contents><Key>key2</Key><LastModified>2017-08-29T15:40:43.269Z</LastModified><ETag></ETag><Size>26847</Size><StorageClass>STANDARD</StorageClass></Contents></ListBucketResult>
+ 
+# curl -i -X GET http://localhost:39999/api/v1/s3/testbucket\?max-keys\=2\&continuation-token\=key3
+HTTP/1.1 200 OK
+Date: Tue, 29 Aug 2017 22:41:18 GMT
+Content-Type: application/xml
+Content-Length: 540
+Server: Jetty(9.2.z-SNAPSHOT)
+ 
+<ListBucketResult xmlns=""><Name>/testbucket</Name><Prefix/><ContinuationToken>key3</ContinuationToken><NextContinuationToken/><KeyCount>2</KeyCount><MaxKeys>2</MaxKeys><IsTruncated>false</IsTruncated><Contents><Key>key3</Key><LastModified>2017-08-29T15:40:44.002Z</LastModified><ETag></ETag><Size>26847</Size><StorageClass>STANDARD</StorageClass></Contents><Contents><Key>testobject</Key><LastModified>2017-08-29T15:36:03.613Z</LastModified><ETag></ETag><Size>26847</Size><StorageClass>STANDARD</StorageClass></Contents></ListBucketResult>
+```
+
+你还可以验证这些对象是否为Alluxio文件，在`/testbucket`目录下。
+
+```bash
+./bin/alluxio fs ls -R /testbucket
+```
+
+### 删除objects
+
+```bash
+# curl -i -X DELETE http://localhost:39999/api/v1/s3/testbucket/key1
+HTTP/1.1 204 No Content
+Date: Tue, 29 Aug 2017 22:43:22 GMT
+Server: Jetty(9.2.z-SNAPSHOT)
+```
+ 
+```bash
+# curl -i -X DELETE http://localhost:39999/api/v1/s3/testbucket/key2
+# curl -i -X DELETE http://localhost:39999/api/v1/s3/testbucket/key3
+# curl -i -X DELETE http://localhost:39999/api/v1/s3/testbucket/testobject
+```
+
+### 初始化multipart upload
+
+```bash
+# curl -i -X POST http://localhost:39999/api/v1/s3/testbucket/testobject?uploads
+HTTP/1.1 200 OK
+Date: Tue, 29 Aug 2017 22:43:22 GMT
+Content-Length: 197
+Server: Jetty(9.2.z-SNAPSHOT)
+
+<?xml version="1.0" encoding="UTF-8"?>
+<InitiateMultipartUploadResult xmlns="">
+  <Bucket>testbucket</Bucket>
+  <Key>testobject</Key>
+  <UploadId>2</UploadId>
+</InitiateMultipartUploadResult>
+```
+
+### Upload part
+
+```bash
+# curl -i -X PUT http://localhost:39999/api/v1/s3/testbucket/testobject?partNumber=1&uploadId=2
+HTTP/1.1 200 OK
+Date: Tue, 29 Aug 2017 22:43:22 GMT
+ETag: "b54357faf0632cce46e942fa68356b38"
+Server: Jetty(9.2.z-SNAPSHOT)
+``` 
+
+### List parts
```

Comment:
translate

Reply:
The file has been updated. Thanks for the comment.